content/pt-br/docs/concepts/scheduling-eviction/_index.md

```diff
@@ -1,8 +1,30 @@
 ---
-title: "Escalonamento"
-weight: 90
+title: "Escalonamento, preempção e expulsão"
+weight: 95
+content_type: concept
 description: >
-   No Kubernetes, agendamento refere-se a garantia de que os pods correspondam aos nós para que o kubelet possa executá-los.
-   Remoção é o processo de falha proativa de um ou mais pods em nós com falta de recursos.
+  No Kubernetes, escalonamento refere-se a garantia de que os Pods correspondam aos Nós para que o kubelet possa executá-los. Preempção é o processo de finalizar Pods com menor prioridade, para que os Pods com maior prioridade possam ser escalonados nos Nós. Expulsão é o processo de finalização proativa de um ou mais Pods em Nós com poucos recursos.
```

Comment:
nit: I know the original page had Node with a capital N here, but translated I think it looks better with a small "n" (nós).

Reply:
Suggestion: use "remoção" instead of "expulsão" (if I'm not mistaken, there are more pages using "remoção" for eviction).

Reply:
Done!